src/pattern.py

```diff
@@ -1,2 +1,9 @@
 
 # Print the pattern
+j = [ ]
+for i in range(1,6):
+    j.append(str("*"))
+    print(" ".join(j))
+for i in range(1,6):
+    j.remove(str("*"))
```

Comment:
For reasons we haven't talked about, `remove` is a slow operation. You can, instead, remove the last element with `j.pop()`.

Reply:
Okey, thank you!